src/IQFeed.CSharpApiClient/Streaming/Level1/Dynamic/Messages/IUpdateSummaryDynamicMessage.cs

```diff
@@ -73,5 +73,7 @@ public interface IUpdateSummaryDynamicMessage
         string Type { get; }
         double Volatility { get; }
         double VWAP { get; }
+        int MostRecentTradeAggressor { get; }
```

Comment:
It doesn't really matter, but I'd add these fields higher in the order how they appear in the IQFeed documentation. 
It will simplify if we ever want to check for discrepances.

Reply:
Yup. At one point I was keeping 6.1 stuff separate, but there's absolutely no point in that anymore. So I'll move them into place.

Reply:
Hmmm, OK, so the stuff in that interface isn't in the order of the documentation anyway. The doc is in alphabetical order now, because of dynamic fields, and this interface is still in the original order given in earlier protocols.
So if I'm going to put them higher up in order, then I should probably reorder the entire interface. Yes?
If Symbol is not the first field, will that screw things up?

Reply:
OK, so I reordered it all into alphabetical order and it seems good. Passes all the tests, anyway.

Reply:
Fields order in the interface doesn't matter. We just need to ensure we put Symbol first when specifying the needed fields and we have a check in place to disallow anything other than Symbol for the first one. :)